### DISCUSSION TITLE
Value and Recognizes Digi ROIs different

### DISCUSSION THREAD

**Discussion by RaRo1980** [jomjol/AI-on-the-edge-device#2816] [Q&A]
Have always problems that i get wrong Value 

In Recognizes is 261.442 but vale is 1000 more. Who i can fox it 
Thanks for your helps ;)

![image](https://github.com/jomjol/AI-on-the-edge-device/assets/66684606/514c8f53-e1a8-4248-af93-fb0f5dd5dc0f)

**Comment by caco3** [comment]
this is a general issue on these meters. you need to set max rate to eg. 0.05 to prevent this.
